api/pkg/db/migration/202107131802_remove_existing_categories.go

```diff
@@ -28,7 +27,7 @@ func removeExistingCategories(log *log.Logger) *gormigrate.Migration {
 	return &gormigrate.Migration{
 		ID: "202107131802_remove_existing_categories",
 		Migrate: func(db *gorm.DB) error {
-			if err := db.Unscoped().Where("name IN ?", []string{"Deploy", "Editor", "Language", "Notification", "Others", "Test Framework"}).Delete(&model.Category{}).Error; err != nil {
+			if err := db.Exec("DELETE FROM categories").Error; err != nil {
```

Comment:
I think we can remove this migration now. as we are updating the config refresh api to handle deletion
??

Reply:
But before running the api we need to fix the categories as well right ??

Reply:
to avoid deletion of all records which are not necessarily to be deleted we are updating the config refresh api
so refreshall migration will add those back in db
and when api server is started the new will get added the ones removed from config will get deleted

Reply:
Yeah agreed on that but ones removed from config which are supposed to be deleted needs to be handled at the api level itself then right